### PR TITLE
add scoreboard export

### PR DESCRIPTION
**Pull request by ghost**
## Related Tickets
- ticket redmine: https://edu-redmine.sun-asterisk.vn/issues/76129

## WHAT (optional)
- Add excel export for scoreboard

## HOW
- I use xlsx npm package for this feature for both student and teacher.

## Evidence (Screenshot or Video)
image
image
image
image

### CODE REVIEW

**Comment by ghost** [comment]
ready

File: src/public/scripts/scoreboard/student-scoreboard.js [hunk 1]
```diff
@@ -19,6 +19,41 @@ $(document).ready(function () {
...
+        ws[cellAddress] = {};
+      }
+      ws[cellAddress].s = {
+        alignment: {
+          horizontal: "right",
+        },
+      };
+    }
+
+    console.log(ws)
```
Comment (huongnt-2545) [inline src/public/scripts/scoreboard/student-scoreboard.js]: xóa debug

File: src/public/scripts/scoreboard/scoreboard.js [hunk 2]
```diff
@@ -86,6 +86,142 @@ $(document).ready(function () {
...
+      { wch: 8 },
+      { wch: 8 },
+      { wch: 8 },
+      { wch: 8 },
+      { wch: 10 },
+      { wch: 10 },
+      { wch: 10 },
+    ];
+
+    console.log(scoreSheet);
```
Comment (huongnt-2545) [inline src/public/scripts/scoreboard/scoreboard.js]: xóa debug

File: src/public/scripts/scoreboard/scoreboard.js [hunk 3]
```diff
@@ -86,6 +86,142 @@ $(document).ready(function () {
...
+      "Test Scores",
+      "",
+      "",
+      "",
+      "",
+      "",
+      "",
+      "",
+      "",
+      "",
```
Comment (huongnt-2545) [inline src/public/scripts/scoreboard/scoreboard.js:117]: title header này là gì e ơi?

Comment (ghost) [reply src/public/scripts/scoreboard/scoreboard.js:117]: cái này do em gộp ô nên em để title rỗng á chị ơi

**Comment by ghost** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]
merged

**Pull request merged**